### ISSUE TITLE
About pretrained caffe model

### ISSUE THREAD

**Issue by Syrup274** [sunnyxiaohu/R-C3D.pytorch#5]
I want to use pytorch pre-trained weights(ucf101-caffe.pth),  you mentioned that:

> If you want to use pytorch pre-trained models, please remember to transpose images from BGR to RGB, and also use the same data transformer (minus mean and normalize) as used in pretrained model.

Can you tell me some details about them? i.e. the mean value you used to minus, and the scale you normalize(-[1,1] or [0,1]), did you use opencv to normalize(or just minus something)?

Thank you!

**Comment by sunnyxiaohu** [comment]
All you have to do is to set pretrained=True (e.g. "tdcnn_demo = c3d_tdcnn(class_agnostic=cfg.AGNOSTIC, pretrained=True)").
The video frames will subtract the mean value  as configured in https://github.com/sunnyxiaohu/R-C3D.pytorch/blob/master/lib/model/utils/config.py

Thanks.
